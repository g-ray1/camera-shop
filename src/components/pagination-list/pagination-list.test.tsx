import { render } from '@testing-library/react';
import { BrowserRouter } from 'react-router-dom';
import PaginationList from './pagination-list';

const handleClick = jest.fn();

describe('PaginationList component', () => {
  it('Snapshot', () => {
    const view = render(<PaginationList cardsCount={10} currentPage={1} clickHandler={handleClick} />, {wrapper: BrowserRouter});

    expect(view).toMatchSnapshot();
  });
});
