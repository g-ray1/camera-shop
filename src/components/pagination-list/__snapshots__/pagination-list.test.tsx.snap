// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PaginationList component Snapshot 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="pagination"
      >
        <ul
          class="pagination__list"
        >
          <li
            class="pagination__item"
          >
            <a
              class="pagination__link pagination__link--active"
              href="/"
            >
              1
            </a>
          </li>
          <li
            class="pagination__item"
          >
            <a
              class="pagination__link "
              href="/"
            >
              2
            </a>
          </li>
          <li
            class="pagination__item"
          >
            <a
              class="pagination__link pagination__link--text"
              href="/"
            >
              Далее
            </a>
          </li>
        </ul>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="pagination"
    >
      <ul
        class="pagination__list"
      >
        <li
          class="pagination__item"
        >
          <a
            class="pagination__link pagination__link--active"
            href="/"
          >
            1
          </a>
        </li>
        <li
          class="pagination__item"
        >
          <a
            class="pagination__link "
            href="/"
          >
            2
          </a>
        </li>
        <li
          class="pagination__item"
        >
          <a
            class="pagination__link pagination__link--text"
            href="/"
          >
            Далее
          </a>
        </li>
      </ul>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
